dfaker.py

```diff
@@ -354,9 +354,20 @@ def normal_bolus(value, timestamp, carb_ratio, params):
 	bolus_entry["subType"] = "normal"
 	insulin = round_to(int(value) / carb_ratio)
 	bolus_entry["normal"] = insulin
+
+	interrupt = random.randint(0,9) #interrupt 1 in 10 boluses
+	if interrupt == 1:
+		bolus_entry = interrupted_bolus(insulin, timestamp, params)
 	dfaker.append(bolus_entry)
 	return bolus_entry
 
+def interrupted_bolus(value, timestamp, params):
+	bolus_entry = {}
+	bolus_entry = add_common_fields('bolus', bolus_entry, timestamp, params)
+	bolus_entry["expectedNormal"] = value
+	bolus_entry["normal"] = value - random.uniform(0, value)
```

Comment:
Round the resulting values for `normal` here like we do for other fields, please.